pkg/sdk/application.go

```diff
@@ -0,0 +1,37 @@
+package sdk
+
+import (
+	"context"
+
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
+)
+
+// ApplicationSuppliers is the structure that represents a session's end block height
+// and its matching suppliers.
+type ApplicationSuppliers struct {
+	// Application is the fully hydrated session object returned by the query.
+	Application []apptypes.Application
+}
+
+// GetApplicationSupplierEndpoints returns a flattened structure of the endpoints
```

Comment:
I don't think this comment reflects the purpose of the function. Please update it.